config/config.go

```diff
@@ -102,21 +120,16 @@ func Configure() (conf Config, err error) {
 		conf = Config{YamlConfig: yconf}
 	}
 
-	conf.BackendURLs = make([]*url.URL, 0, len(conf.Backends))
-	for _, rawstr := range conf.Backends {
-		url, perr := url.Parse(rawstr)
-		if perr != nil {
-			return conf, perr
-		}
-		conf.BackendURLs = append(conf.BackendURLs, url)
-	}
-
-	if len(conf.MaintainedBackend) > 0 {
-		u, perr := url.Parse(conf.MaintainedBackend)
-		if perr != nil {
-			return conf, perr
+	confFile, openErr := os.Open(*confFilePath)
+	if openErr == nil {
+		yconf, parseErr := parseConf(confFile)
+		if parseErr != nil {
+			return conf, parseErr
 		}
-		conf.MaintainedBackend = u.Host
+		conf = Config{YamlConfig: yconf}
+	} else {
+		fmt.Println("Cannot read config file:", openErr.Error())
```

Comment:
I'll remove above print. I would rather use log.Fatal, but as all loggers log to syslog error message would not be visible to user. So I'll leave fmt.Print in main.go and return in main func